operators/pkg/controller/elasticsearch/driver/esstate.go

```diff
@@ -0,0 +1,129 @@
+package driver
```

Comment:
license header should be on top?

Reply:
Yes! I'll merge that one and fix it in #1262.